model/pytorch/dffml_model_pytorch/pytorch_base.py

```diff
@@ -63,6 +63,14 @@ def __post_init__(self):
                 map(self.clstype, self.classifications)
             )
 
+class LossFunctionNotFoundError(Exception):
+    """
+    Exception raised when loss function is misspelled.
+    """
+    def __init__(self, message="Loss function is not valid!"):
```

Comment:
Not sure if we need this `__init__`. It would probably be sufficient to just pass this error message every time. We should probably say what the spelling they gave was, if possible.